src/Http/Response/Factory.php

```diff
@@ -7,8 +7,8 @@
 use Illuminate\Support\Str;
 use Dingo\Api\Http\Response;
 use Illuminate\Support\Collection;
-use Illuminate\Pagination\Paginator;
 use Dingo\Api\Transformer\Factory as TransformerFactory;
+use Illuminate\Contracts\Pagination\LengthAwarePaginator;
```

Comment:
LOL.. I totally missed that the interface extends the other, Sorry about that.

All good then :+1: 

I will change it to the Paginator contract